src/app/organizations/settings/organization-subscription.component.ts

```diff
@@ -205,13 +216,23 @@ export class OrganizationSubscriptionComponent implements OnInit {
   }
 
   get subscriptionMarkedForCancel() {
+    console.log(`Marked for cancellation: ${this.subscription != null && !this.subscription.cancelled && this.subscription.cancelAtEndDate}`);
```

Comment:
Looks like this guy got left in and the lint check failed.

Reply:
Prettier might also needs to be run based on the build check logs

Reply:
Ouch! Removed ...

Reply:
Thank you! I think you need to run prettier (`npm run prettier`) as there are these linting errors in our checks still:
![Screen Shot 2022-02-02 at 12 59 53 PM](https://user-images.githubusercontent.com/8926729/152211209-e46c24d8-2c41-4d05-ad27-352ff7577a9d.png)